src/journal.c

```diff
@@ -103,10 +115,10 @@ SDB_API bool sdb_journal_clear(Sdb *s) {
 }
 
 SDB_API bool sdb_journal_unlink(Sdb *s) {
-	const char *filename = sdb_journal_filename (s);
-	sdb_journal_close (s);
-	if (filename) {
-		return !unlink (filename);
+	char filename[256];
```

Comment:
```suggestion
	char filename[SDB_MAX_PATH];
```

which should be the same as MAX_PATH from limits.h, or 256 if not defined (windows)